packages/test-utils/src/create-dom-event.js

```diff
@@ -35,14 +72,12 @@ function createEvent(
       ? window[eventInterface]
       : window.Event
 
-  const event = new SupportedEventInterface(type, {
+  const event = new SupportedEventInterface(
+    type,
     // event properties can only be added when the event is instantiated
     // custom properties must be added after the event has been instantiated
-    ...options,
-    bubbles,
-    cancelable,
-    keyCode: modifiers[modifier]
-  })
+    sanitizeOptions(type, modifier, { bubbles, cancelable }, options)
```

Comment:
Ah. I think the cheapest fix for this would be the following, which keeps feature parity and will let the modifier win over the code.

```js
{
    ...options,
    bubbles,
    cancelable,
    keyCode: modifiers[modifer] || options.keyCode || options.code
}
```